run_analysis.sh

```diff
@@ -13,25 +21,27 @@ done
 # Clean-up workspace
 #./utils/clean.sh
 
-# Parse arguments
-ERA=$1               # options: 2016, 2017
-CHANNELS=${@:2}      # options: em, et, mt, tt
 
 # Create shapes of systematics
-./shapes/produce_shapes.sh $ERA $CHANNELS
+#./shapes/produce_shapes.sh $ERA $CHANNELSARG $METHOD
 
 # Apply blinding strategy
 #./shapes/apply_blinding.sh $ERA
 
 # Convert shapes to synced format
-./shapes/convert_to_synced_shapes.sh $ERA
+for CHANNEL in ${CHANNELS[@]}; do
+    logandrun ./shapes/convert_to_synced_shapes.sh $ERA $CHANNEL $METHOD &
```

Comment:
The logandrun will not catch the error if the input file for ./shapes/convert_to_synced_shapes.sh does not exist (it printed "COMPLETE" and continued (i mean, it's not worse than before..)